README.md

```diff
@@ -116,6 +116,7 @@ Check out my [blog](https://nikolaskama.me/) and follow me on [Twitter](https://
 * [Termite](https://github.com/thestinger/termite/) - A keyboard-centric terminal, aimed at use within a tiling window manager.
 * [Cmder](https://github.com/cmderdev/cmder) - Lovely console emulator package for Windows.
 * [Terminix](https://gnunn1.github.io/terminix-web/) - An advanced GTK3 tiling terminal emulator.
+* [Yakuake](https://yakuake.kde.org/) - Yakuake is a drop-down terminal emulator based on Konsole.
```

Comment:
Please change `Yakuake is a drop-down terminal emulator based on Konsole.` to `Drop-down terminal emulator based on Konsole.` Thanks in advance!